File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.java
class Solution {
    public int removeDuplicates(int[] arr) {
        int n = arr.length;
        int i =0;
        int j =0;
        while(j<n){
            if(arr[i]==arr[j]){
                j++;
            }
            else if(arr[i]!=arr[j]){
                arr[i+1] = arr[j];
                i++;
            }
        }

        return i+1;
    }
}

File: Data Structures & Algorithms/can-place-flowers/submission-0.py
class Solution:
    def canPlaceFlowers(self, arr: List[int], n: int) -> bool:
        count = 0
        for i in range(1,len(arr)-1):
            if arr[i-1]==0 and arr[i+1]==0:
                count+=1
        
        if count>=n:
            return True
        else:
            return False

